OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -229,29 +229,48 @@ public SlotDropDownOption(string title, string order, Func<bool> selected)
 
 		public static void SelectSpawnPoint(OrderManager orderManager, MapPreviewWidget mapPreview, MapPreview preview, MouseInput mi)
 		{
```

Comment:
We will need a ready check here:
```csharp
if (orderManager.LocalClient.State == Session.ClientState.Ready)
	return;
```

to avoid a bug:
* Create a lobby with more spawns disabled than there are players (so the game doesn't autostart at the next step)
* Check the ready checkbox next to your name in the player list
* Right click on an enabled spawn

Expected behaviour: Nothing happens
Actual behaviour: A pair of server error messages related to setting state while ready

The check in `SelectPlayerSpawnPoint` can then be removed, and the observer part of that was wrong!